test/factories/host_related.rb

```diff
@@ -48,7 +61,7 @@ def set_nic_attributes(host, attributes, evaluator)
   factory :nic_base, :class => Nic::Base do
     type 'Nic::Base'
     sequence(:identifier) { |n| "eth#{n}" }
-    sequence(:mac) { |n| "00:00:00:00:" + n.to_s(16).rjust(4, '0').insert(2, ':') }
```

Comment:
I don't think they have any relevance to Rails 4, you're right. I included them as they were a compatible part of https://github.com/theforeman/foreman/pull/2055